deploy.sh

```diff
@@ -11,4 +11,8 @@ mkdir -p ~/.ssh && chmod 0700 ~/.ssh
 cat ${GITHUB_ACTION_PATH}/known_hosts >> ~/.ssh/known_hosts
 
 platform project:set-remote ${PLATFORM_PROJECT_ID}
-platform push -vv --activate --force --target ${GITHUB_REF_NAME}
+if [[ -n "$FORCE_PUSH" ]]; then
+  platform push -vv --activate --force --target ${GITHUB_REF_NAME}
+else 
+  platform push -vv --activate --target ${GITHUB_REF_NAME}
+fi
```

Comment:
```suggestion
PLATFORM_OPTS="-vv --activate --target ${GITHUB_REF_NAME}"
if [[ -n "$FORCE_PUSH" ]]; then
  PLATFORM_OPTS="$PLATFORM_OPTS --force"
fi
platform push ${PLATFORM_OPTS}
```

What do you think about this as an alternative?